src/main.rs

```diff
@@ -305,6 +313,19 @@ fn main() -> Result<(), CompileError> {
             print!("{}", the_json.unwrap());
             print_time = false;
         }
+        Args::MakeParametersList(clist) => {
+            let constants_map =
+                make_parameters_list(clist.consts_file.as_ref().map(|s| Path::new(s))).unwrap();
+            match serde_json::to_string(&constants_map) {
+                Ok(s) => {
+                    println!("{}", s);
+                }
+                Err(e) => {
+                    panic!("{}", e);
```

Comment:
If you're going to panic with the error, you might as well call unwrap on the result.